recipe/meta.yaml

```
@@ -20,21 +20,20 @@ requirements:
     - wheel
   run:
     - python
-    - bokeh >=2.4.2,<3.0
-    - cytoolz >=0.8.2
+    - bokeh >=2.4.2
```

```suggestion
    - bokeh >=2.4.2,<3.0
```
Because of api changes in bokeh 3